src/locales/ar/translation.json

```diff
@@ -0,0 +1,47 @@
+{
+  "title": "[ar] Project Title",
+  "banner": "[ar] An official website of New York City government",
+  "nav": {
+    "dropdown": [],
+    "dropdownOne": {
+      "label": "[ar] Nav Label 1",
+      "simpleLinkOne": "[ar] Simple link one",
+      "simpleLinkTwo": "[ar] Simple link two"
+    },
+    "dropdownTwo": {
+      "label": "[ar] Nav Label 2",
+      "simpleLinkThree": "[ar] Simple link three",
+      "simpleLinkFour": "[ar] Simple link four"
+    },
+    "parentOne": "[ar] Parent link"
+  },
+  "secondaryLink": [],
+  "secondaryLinks": {
+    "linkOne": "[ar] Link one",
+    "linkTwo": "[ar] Link two"
+  },
+  "footer": {
+    "copyright": "[ar] © 2021 City of New York. All Rights Reserved.",
+    "trademark": "[ar] NYC is a trademark and service mark of the City of New York.",
+    "terms": "[ar] NYC Terms of Use",
+    "privacy": "[ar] NYC Privacy Policy",
+    "primary": [
+      "Primary Link 1",
```

Comment:
Lines 29-32 need an `[ar]`

Reply:
In the future you can just suggest a change on 1 translation file, no need to comment on all 13. Same goes for the markdown files :)